tests/test_cases/test_struct/test_struct.py

```diff
@@ -25,7 +25,10 @@
 )
 async def test_packed_struct_format(dut):
     """Test that the correct objects are returned for a struct"""
-    assert repr(dut.my_struct) == "LogicObject(sample_module.my_struct)"
+    if SIM_NAME.startswith("verilator"):
+        assert repr(dut.my_struct) == "LogicObject(sample_module.my_struct)"
```

Comment:
Hmmm... Can we not tell the object is a packed object, albeit with no subfields accessible?

Reply:
Not sure, I think we would have to indicate in https://github.com/verilator/verilator/blob/master/include/verilated_vpi.cpp, but even then I'm not sure if verilator includes info like that after verilation, so the marker might have to be put in earlier in the process